src/pretix/control/templates/pretixcontrol/vouchers/bulk.html

```diff
@@ -78,6 +78,8 @@ <h1>{% trans "Create multiple vouchers" %}</h1>
             {% bootstrap_field form.tag layout="control" %}
             {% bootstrap_field form.comment layout="control" %}
             {% bootstrap_field form.show_hidden_items layout="control" %}
+            {% bootstrap_field form.all_addons_included layout="control" %}
+            {% bootstrap_field form.all_bundles_included layout="control" %}
```

Comment:
Somehow it feels more natural to me to place this somewhere around price change, e.g. before `{% if form.subevent %}`. Although one needs to remove the margin-bottom from the `div.alert`

![Bildschirmfoto 2023-05-16 um 22 33 41](https://github.com/pretix/pretix/assets/276509/1d5c7eb6-c195-410f-a4f0-d289cb199b24)